gst/nnstreamer/include/nnstreamer_plugin_api_util.h

```diff
@@ -435,5 +435,12 @@ nnstreamer_version_string (void);
 extern void
 nnstreamer_version_fetch (guint * major, guint * minor, guint * micro);
 
+/**
+ * @brief Compare dimension strings
+ * @return TRUE if equal, FALSE if given dimension strings are invalid or not equal.
+ */
+extern gboolean
+gst_tensors_dimension_string_equal (const gchar * dimstr1, const gchar * dimstr2);
```

Comment:
function name: gst_tensor_dimension_string_is_equal() and move this declaration to line 302

Reply:
Updated! 
Thank you :)